src/commands/Moderation/verify.js

```diff
@@ -0,0 +1,62 @@
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('verify')
+        .setDescription('Verify in the staff server.')
+        .addStringOption(option => 
+            option.setName('position')
+                .setDescription('Please enter the position you applied for.')
+                .setRequired(true)
+        ),
+        async execute(interaction) {
+            try {
+                const appliedRole = interaction.options.getString('position');
+        
+                const roleReply = await interaction.reply(`<@${interaction.user.id}>, you have requested verification for the role: **${appliedRole}**. Please provide proof of your application by sending an image.`);
+        
+                const proofCollector = interaction.channel.createMessageCollector({
+                    filter: msg => {
+                        if (msg.attachments.size > 0 && msg.attachments.every(attachment => attachment.contentType.startsWith('image'))) {

```

Comment:
Could you check the send to check if they are the interaction user.

Reply:
Phrased that horribly. When it recieves the callback you should confirm it's from the original sender.